app/connector/rest-swagger/pom.xml

```diff
@@ -44,6 +43,16 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.syndesis.common</groupId>
+      <artifactId>common-util</artifactId>
```

Comment:
Not too sure about adding this dependency

Reply:
PMD was complaining